tests/unit/test_browse.py

```diff
@@ -553,18 +604,38 @@ def test_convert_4_multiband_masked_to_raster(self):
         )
         ds.to_numpy.return_value = input_array
 
-        expected_raster = np.ma.array(
-            data=[
-                [[0, 0, 0, 121], [0, 0, 0, 64], [0, 0, 255, 0], [0, 0, 13, 255]],
-                [[0, 0, 0, 255], [0, 0, 0, 255], [0, 0, 255, 255], [0, 0, 255, 255]],
-                [[0, 0, 0, 121], [0, 0, 0, 64], [0, 0, 255, 0], [0, 0, 13, 255]],
-                [[0, 0, 0, 255], [0, 0, 0, 255], [0, 0, 255, 255], [0, 0, 255, 255]],
+        expected_raster = np.array(
```

Comment:
These values are no longer scaled.

Reply:
Well, yes and no.  Yes, they are all in the valid range so they aren't scaled, but they're no longer scaled, because the code before this change did scale all 4 band rasters https://github.com/nasa/harmony-browse-image-generator/blob/2.0.2/hybig/browse.py#L237-L243